molecule/external-activemq/prepare.yml

```diff
@@ -0,0 +1,55 @@
+---
+- name: Default prepare
+  import_playbook: ../default/prepare.yml
+
+- name: Prepare activemq instance
+  hosts: localhost
+  gather_facts: false
+  vars:
+    broker_name: molecule-{{ lookup('env', 'BUILD_NUMBER') }}
+  tasks:
+    - name: Create EC2 security group for mq broker
+      amazon.aws.ec2_group:
+        name: "{{ broker_name }}"
+        description: Molecule external activemq EC2 test
+        rules:
+          - proto: tcp
+            from_port: 61617
+            to_port: 61617
+            cidr_ip: "{{ ansible_ec2_local_ipv4 }}/32"
```

Comment:
In that case nginx and repo are the same but that's generally not true. At least comment the assumption or state this won't work with multimachines. Otherwise using the full subnet is not a problem imho

Reply:
correct, but since this scenario cover just a single instance it should be good to go

or maybe we can cover the external activemq in the multi-node scenario instead of using this dedicated scenario? 🤔

Reply:
that would actually make sense to me. Up to you if you prefer merging as is it's fine too.